.github/workflows/reusable-test.yaml

```diff
@@ -147,3 +147,19 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: ./logs
+
+  codespell:
+    name: Check spelling
+    runs-on: ubuntu-latest
+  steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v4
+
+    - name: Setup environment
+      uses: ./.github/workflows/actions/setup-environment
+
+    - name: Install dependencies
+      uses: ./.github/workflows/actions/install-dependencies
+
+    - name: Spell Check
+      run: pnpm cspell --config ./cspell.json
```

Comment:
It looks like the indentation here is not correct, breaking the workflow https://github.com/LayerZero-Labs/devtools/actions/runs/8542631863